.github/workflows/release.yaml

```
@@ -256,6 +256,42 @@ jobs:
           signing-key: ${{ secrets.COSIGN_PRIVATE_KEY }}
           signing-password: ${{ secrets.COSIGN_PASSWORD }}
 
+  build-gcr-deployer:
+    name: Build GCR deployer image
+    environment: Release
+    runs-on: ubuntu-latest
+    needs: [ prepare ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Login to GCR
+        uses: redhat-actions/podman-login@4934294ad0449894bcd1e9f191899d7292469603 # v1.7
+        with:
+          registry: gcr.io
+          username: ${{ secrets.GCR_USERNAME }}
+          password: ${{ secrets.GCR_JSON_KEY }}
+      - name: Build target
+        id: build-image
+        uses: redhat-actions/buildah-build@7a95fa7ee0f02d552a32753e7414641a04307056 # v2.13
+        with:
+          oci: true
+          context: ./config/helm/
+          containerfiles: |
+            ./config/helm/Dockerfile
+          tags: ${{ needs.prepare.outputs.version }}
```

This version includes a leading "v", the deployer image in our old pipeline was released without a "v"

---

another idea can be: 
```suggestion
          tags: ${{ needs.prepare.outputs.version_without_prefix }}  ${{ needs.prepare.outputs.version }} 
```
add both, or I can just use only without prefix.

---

fixed in 9d68f27b6

---

Can you also add the `major.minior` tag

---

done
```
crane manifest gcr.io/dynatrace-marketplace-dev/dynatrace-operator/deployer:1.5
crane manifest gcr.io/dynatrace-marketplace-dev/dynatrace-operator/deployer:1.5.8
```